.github/workflows/raydp.yml

```diff
@@ -19,9 +19,9 @@ name: RayDP CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ main, master ]
```

Comment:
It seems we don't have the main branch.

Reply:
right, i added it b/c main is now the default branch rather than master.  that way the workflow runs:  https://github.com/mjschock/raydp/actions/runs/1007895832